src/components/Confirm/Previews.js

```diff
@@ -124,7 +124,12 @@ const Previews = localised(
               }}
             />
           ) : (
-            <PageTitle title={title} smaller={true} className={style.title} />
+            <PageTitle
+              title={title}
+              smaller={true}
+              className={style.title}
+              shouldAutoFocus={methodNamespace !== 'doc_confirmation'}
```

Comment:
Does this mean that for documents the confirmation screen won't be announced?

Reply:
This only affects the focused element to not be the screen (page) title like for this case where on closing the preview we want the focus to start from the `Enlarge image` button. Title is always announced as the component's `aria-live` is set to "assertive"